app/admin/projects.rb

```diff
@@ -58,9 +58,21 @@
       column(:counterpart)
       column :created_at
     end
+    h1 'Counterparts:'
+    table_for project.counterparts do
+      column(:name).pluck(:name)
```

Comment:
ça marche ça ? j'avais jamais vu cette façon de faire

quelle différence par rapport à `column(:name)` ?

Reply:
J'avais lu ça en fait :
https://stackoverflow.com/questions/20794398/how-can-i-have-activerecords-pluck-also-return-the-column-name-for-rendering-in

Mais je crois pas que ça s'applique ici finalement. Je l'enlève à la prochaine PR.

Reply:
Yes j'ai l'impression que c'est autre chose